robot_framework/process.py

```diff
@@ -12,20 +12,28 @@
 import openpyxl
 from openpyxl.worksheet.worksheet import Worksheet
 import pyodbc
+import itk_dev_event_log
 
 from robot_framework import config
 
 
 def process(orchestrator_connection: OrchestratorConnection) -> None:
     """Do the primary process of the robot."""
     orchestrator_connection.log_trace("Running process.")
+    event_log = orchestrator_connection.get_constant("Event Log")
+    itk_dev_event_log.setup_logging(event_log.value)
 
     excel_file = get_email_attachment(orchestrator_connection)
     wb = read_excel_file(excel_file)
+    initial_rows = wb.active.max_row
 
     get_address_changes(wb)
+
     calculate_difference(wb)
+    persistent_rows = wb.active.max_row
 
+    itk_dev_event_log.emit(orchestrator_connection.process_name, "Updated address.", initial_rows)
+    itk_dev_event_log.emit(orchestrator_connection.process_name, "Removed address.", initial_rows-persistent_rows)
```

Comment:
```suggestion
    itk_dev_event_log.emit(orchestrator_connection.process_name, "No change to address.", initial_rows-persistent_rows)
```
Does this make more sense?